style.css

```diff
@@ -0,0 +1,446 @@
+@import url("https://fonts.googleapis.com/css2?family=Delicious+Handrawn&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@200;600&display=swap");
+
+body {
+  background-image: url(Bilder/purple_jellyfish.jpg);
+  background-repeat: repeat;
+  background-position: top;
+  background-attachment: fixed;
+  text-align: left;
+  margin: 0;
+}
+
+/*Schriften*/
```

Comment:
Great that you have divided your CSS into these different sections! To make it even more "clean", you could order your headlines in h1-h5 order as well :)